runtime/test/storage-proxy-test.js

```diff
@@ -129,6 +129,9 @@ class TestParticle {
     if ('removed' in update) {
       details += '-' + this._toString(update.removed);
     }
+    if (update.originator) {
+      details += '(originator)';
```

Comment:
Should this use the actual value of originator, so tests can confirm that the id is threaded through correctly? Or would that just be testing the test code?

Reply:
By the time we get here, the value is `true` or `false`

Reply:
Still worth testing that it's the correct value, though?

Reply:
Split this into `originator` (bool) and `originatorId` (particle ID)